### ISSUE TITLE
Pixracer: Establish HW test set

### ISSUE THREAD

**Issue by LorenzMeier** [PX4/Hardware#43]
@pkocmoud @nickarsov Is there something I can help with in terms of the HW test routines? I presume you use something like ```tests sensors``` - what else do you need?

**Comment by proficnc** [comment]
You could look at the test suite that Tridge and I (and others) did for Pixhawk 2, it's all open source.

**Comment by LorenzMeier** [comment]
@proficnc Indeed. @pkocmoud I've followed the test jig development quite closely, let me know when you want to get up to speed and sync.

@kd0aij @pkocmoud I've fixed the 'tests' command (stack) and it should behave now on master for you.
